map/map_loader/config/lanelet2_map_loader.param.yaml

```diff
@@ -1,4 +1,5 @@
 /**:
   ros__parameters:
+    allow_unsupported_version: false # flag to load unsupported format_version anyway. If true, just prints warning.
```

Comment:
Maybe we could first introduce with true as default so that people would have time to fix their map from the warning output. Also, we should define about the version control and announce them before making false as default. (e.g. announce rules about what change would increase the minor/major version number)

Reply:
fixed in ef82188daad2e113d10c3090a3f347a0cdc7dc85
This parameter is true in the autoware_launch PR, just a mistake